script/release-prepare.sh

```diff
@@ -60,11 +60,13 @@ else
     body=''
 fi
 
-body=$(cat "$RELEASE_NOTES")
-
 # Bump the version
 "$DIR"/version-bump.sh
 
+DEV_VERSION=$("$DIR"/get-version.sh)
+git add --update
```

Comment:
For my understanding, what file(s) does this update?
Should we maybe add a comment?

Reply:
05e01a929a5e6b4ce67908070cd04496c885889c